### PR TITLE
feat(azure-apps): Update azureKvCsiProvider

### LINKED ISSUE #710: Version 1.2.0 of azureKvCsiProvider is available!
Hi!<br/></br>There is a new version available for azureKvCsiProvider!<br/><br/>current targetRevision set: `1.1.*`<br/>new version available: `1.2.0`<br/><br/>Please consider updating the `targetRevision` with a PR in this file [values.yaml](https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/azure-apps/values.yaml)<br/><br/>Thanks.

### LINKED ISSUE #786: Version 1.3.0 of azureKvCsiProvider is available!
Hi!<br/></br>There is a new version available for azureKvCsiProvider!<br/><br/>current targetRevision set: `1.1.*`<br/>new version available: `1.3.0`<br/><br/>Please consider updating the `targetRevision` with a PR in this file [values.yaml](https://github.com/adfinis/helm-charts/tree/master/charts/azure-apps/values.yaml)<br/><br/>Thanks.

### PR DESCRIPTION
**Pull request by gianklug**
# Description

Updated azureKvCsiProvider Version to ~~1.2.0~~ 1.3.0

# Issues

* fixes #710
* fixes #786

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by hairmare** [comment]
please update the description to match the template, thx

**Comment by eyenx** [comment]
As good point to start, check https://github.com/adfinis/helm-charts/pull/806

This is the bare minimum we need in a PR to be able to move forward.

**Comment by hairmare** [comment]
needs a smol rebase and is good to go afterwards

let's chat about getting it rolled out where/if it makes sense afterwards (we'll want/have to take care of #802 in tandem with this)

**Comment by eyenx** [comment]
bump @giank

File: charts/azure-apps/values.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ azureKvCsiProvider:
   # -- Chart
   chart: "csi-secrets-store-provider-azure"
   # -- [csi-secrets-store-provider-azure Helm chart](https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/charts/csi-secrets-store-provider-azure) version
-  targetRevision: "1.1.*"
+  targetRevision: "1.3.0"
```
Comment (eyenx) [inline charts/azure-apps/values.yaml:16]:
```suggestion
  targetRevision: "1.3.*"
```

**Comment by gianklug** [comment]
can we merge this now?

**Comment by eyenx** [comment]
put it on non draft and we will review it.

**Review by hairmare — APPROVED** [review]

**Pull request merged**